src/ore/src/tracing.rs

```diff
@@ -141,12 +155,18 @@ pub async fn configure(config: TracingConfig) -> Result<(), anyhow::Error> {
         stack,
         config.opentelemetry_endpoint,
         config.opentelemetry_headers,
+        config.opentelemetry_log_filter,
     )
     .await?;
 
     Ok(())
 }
 
+/// Shutdown any tracing infra, if any.
+pub fn shutdown() {
+    opentelemetry::global::shutdown_tracer_provider();
+}
+
```

Comment:
Can we do without this since we're not using it?

```suggestion
```

Reply:
does this flush out any pending info? my project for friday is to add tracing to the persist open-loop benchmark, which unlike materialized does have a clean shutdown

Reply:
@danhhz that is my understanding, yes!